### ISSUE TITLE
Text density is too low

### ISSUE THREAD

**Issue by jmandel** [zulip/zulip-mobile#3632]
Comparing text display alone (without looking at topic headers, message spacing, etc) Zulip Mobile needs ~33% more vertical space to display a message than Slack. Main factors are font size and line spacing (both larger in Zulip, and neither adjustable). This leads to more scrolling, and less context visible at a time.

Unlike on desktop, where window sizes and zoom levels can be adjusted independent of app settings, the mobile app has fixed behavior.

There are probably various ways to improve (increase) message density, but creating denser text defaults and perhaps adding adjustable settings seems like low hanging fruit. Also shrinking avatar icons (and possibly left margin size) seems feasible.

**Comment by timabbott** [comment]
I agree; I think we should be aiming for a similar density of the text display to Slack, not something dramatically smaller.  And I agree it seems like the big difference is font size and line height, not a more complex element of the visual design.

**Comment by gnprice** [comment]
Interesting observation, thanks!

I haven't compared these layout aspects side-by-side with other apps in a while. I agree that we should be going for similar density to comparable apps, and in fact I'd want to aim for the dense side of the range of variation.

---

Specifically just on the size of avatars, there was some previous discussion on #3515 and then [in chat](https://chat.zulip.org/#narrow/stream/48-mobile/topic/avatar.20layout/near/752153), and we made them bigger a few months ago in @ishammahajan 's #3529. Comparative data [here](https://github.com/zulip/zulip-mobile/issues/3515#issuecomment-500539364). In short, our new larger size is the same as Slack; our older size was the same as FB Messenger and LINE.

The main point of #3529 was to make the avatars bigger in the conversation list -- previously they were much smaller there than comparable apps. We waffled then about whether to make them bigger in the message list too, and I wouldn't mind undoing that part to make them smaller again.

**Comment by jmandel** [comment]
Indeed, I'm thinking here about the message view in particular (which is where I spend most of my time and where seeing more information for context is most important).  FWIW in this view,. Zulip Mobile's gravatar icons are currently larger than Slack Mobile's.

**Comment by gnprice** [comment]
> Indeed, I'm thinking here about the message view in particular (which is where I spend most of my time and where seeing more information for context is most important).

Yeah, agreed re: that being the most important environment for information density. I understood that focus from your issue description; I'll also go edit the title so it's common-knowledge clear that that's the focus of this issue.


> FWIW in this view,. Zulip Mobile's gravatar icons are currently larger than Slack Mobile's.

Huh interesting, thanks. What I found [in June](https://github.com/zulip/zulip-mobile/issues/3515#issuecomment-500539364) was that Slack's appeared to be 48px , and that's the same size we enlarged ours to (from 32px before). Will have to look again.

**Comment by Gautam-Arora24** [comment]
@gnprice  I want to work on this issue.
